docsets/DokuWiki/docset.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "DokuWiki",
-    "version": "3.0 (Frusterick Manners)",
-    "archive": "dokuwiki-docset.tgz",
+    "version": "4.0 (Hogfather)",
+    "archive": "versions/4.0-hogfather/dokuwiki-docset.tgz",
```

Comment:
This doesn't work. The main archive needs to be outside of the specific versions folder.